drivers/hmis/app/graphql/types/hmis_schema/occurrence_point_form.rb

```diff
@@ -27,6 +27,10 @@ def definition(parent:)
       definition
     end
 
+    def data_collected_about
```

Comment:
@gigxz , I'm still a bit confused about what caused the bug. I wasn't able to reproduce it locally, and also noticed that in the spec test, the instance has a nil value for data collected about. https://github.com/greenriver/hmis-warehouse/blob/release-143/drivers/hmis/spec/requests/hmis/enrollment_occurrence_point_spec.rb#L69

Reply:
I can keep investigating before merging this PR if you think it is non-urgent.

Reply:
Enrollment occurrence_point_forms provides a default https://github.com/greenriver/hmis-warehouse/blob/release-143/drivers/hmis/app/models/hmis/hud/enrollment.rb#L360

And so do Client Detail Forms https://github.com/greenriver/hmis-warehouse/blob/release-143/drivers/hmis/app/graphql/types/hmis_schema/query_type.rb#L512

Project occurrence_point_form_instances don't appear to provide a default https://github.com/greenriver/hmis-warehouse/blob/release-143/drivers/hmis/app/models/hmis/hud/project.rb#L297-L309

But this is no longer used from the frontend perspective, and the Sentry error refers to `GetEnrollmentDetails`

Reply:
> Project occurrence_point_form_instances don't appear to provide a default .. But this is no longer used from the frontend, and the Sentry error refers to GetEnrollmentDetails

I see, thanks for that summary. At the time when the issue surfaced, the backend was on release-142 and the frontend was still on release-141. So the frontend was still using the project occurrence point details query in GetEnrollmentDetails. 

So now that the frontend and backends are all on 142, this bug would only be present when resolving `project { occurrencePointForms }`, which we no longer do. It's still part of the API, though. I think its safest to merge this change.

Reply:
Ah, it makes sense that this is related to the backend and frontend being on different releases. I was wondering if it was something like that. I was able to reproduce it and confirm that setting the default value fixes the problem.

Reply:
Is it ok to keep this PR targeted at release-143, or would you like me to open it against release-142?

Reply:
I think 143 is good, thanks!